supervision/README.md

```diff
@@ -0,0 +1,72 @@
+# Supervision Workgroup
+
+
+The goal of the group is improving FreeBSD in the following areas:
+
+- process tracking
+- process resource control
+- process confinment
+- process supervision
+- service management
+- tools and languages for the above
+
+
+## Process Tracking
+
+Process tracking is how do we reliably track the state of the process, such
+that we can always say if the process is up or down (example: linux PID
+namespace, tracked but not confined. cgroups also fit in, if we forget for a
+moment that cgroups are also resource management tools)
```

Comment:
Solaris contracts https://illumos.org/man/5/contract might be worth mentioning, too